src/views/tutorial-view/index.tsx

```diff
@@ -0,0 +1,8 @@
+export default function TutorialView(props) {
```

Comment:
Explicitly not doing prop interfaces here yet until I get a better idea of what is going on the page.